src/Orckestra.Composer.Website/App_Data/Composite/DynamicTypeForms/Orckestra/Composer/CompositeC1/DataTypes/ISearchTerm.xml

```diff
@@ -20,14 +20,14 @@
             <cms:bind source="Value" />
           </TextBox.Text>
         </TextBox>
-        <DoubleKeySelector Label="Website ID" Help="" Required="true">
-          <DoubleKeySelector.FirstKey>
+        <KeySelector OptionsKeyField="Key" OptionsLabelField="Value" Label="Website ID" Help="" Required="true">
+          <KeySelector.Selected>
             <cms:bind source="WebsiteId" />
-          </DoubleKeySelector.FirstKey>
-          <DoubleKeySelector.Options>
-            <f:StaticMethodCall Type="Composite.Plugins.Routing.Hostnames.FormFunctions" Method="GetRootPages" />
-          </DoubleKeySelector.Options>
-        </DoubleKeySelector>
+          </KeySelector.Selected>
+          <KeySelector.Options>
+            <f:StaticMethodCall Type="Orckestra.Composer.Utils.PageHelper" Method="GetWebsitesRootPages" />
```

Comment:
Can you try to use method Composite.Plugins.Functions.WidgetFunctionProviders.StandardWidgetFunctionProvider.DataReference.HomePageSelectorWidgetFunction.GetHomePages instead of creating a new one?